mdoc/Test/en.expected-vbnet/MyFramework.MyNamespace/MyClassExtensions.xml

```diff
@@ -0,0 +1,41 @@
+<Type Name="MyClassExtensions" FullName="MyFramework.MyNamespace.MyClassExtensions">
+  <TypeSignature Language="C#" Value="public static class MyClassExtensions" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi abstract sealed beforefieldinit MyClassExtensions extends System.Object" />
+  <TypeSignature Language="VB.NET" Value="Public Module MyClassExtensions" />
+  <AssemblyInfo>
+    <AssemblyName>DocTest-DropNS-classic</AssemblyName>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>To be added.</summary>
+    <remarks>To be added.</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName="AnExtension">
+      <MemberSignature Language="C#" Value="public static bool AnExtension (this MyFramework.MyNamespace.MyClass value);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig bool AnExtension(class MyFramework.MyNamespace.MyClass value) cil managed" />
+      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xD;&#xA;Public Function AnExtension (value As MyClass) As Boolean" />
```

Comment:
Not sure I like the line break and line feed characters. At a minimum, it makes it windows-specific, since other platforms do not use both of these characters to terminate a line.

I'd like for you to check and see if it would be synctactically valid to have the `Extension` on the same line ... ie. `&lt;Extension()&gt; Public Function AnExtension (value As MyClass) As Boolean`

Reply:
You're right, we can have the `<Extension()>` on the same line, I'll delete line break symbols here. But there is a case when we definitely need the new line symbols:
```
Public Class TestClass 
    Inherits TestClass2
    Implements Interface1, Interface2
```
Otherwise, VB.NET compiler shows the error:
```
BC30205	End of statement expected.
```
Should we use some other line ending symbols?

Reply:
My initial reaction is to simply use [`Environment.NewLine`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=netframework-4.7.1#System_Environment_NewLine), so that at least it respects the conventions of the platform that you're running _mdoc_ on. However ... I suspect there won't be a "good" answer here. 

I'd like to get at least one other opinion on the matter. @jonpryor, @conceptdev, @lobrien, @dend ... Any thoughts on what approach we should take to newline char(s) in VB signatures?

Reply:
My hot-take is “The values of attributes in the ecmadoc must be idempotent / referentially transparent / deterministic. So therefore, Env.NewLine is actually _not_ the correct answer.”

Reply:
Another idea that @lobrien had, was to potentially use a character like `¶` (which is encoded in the XML as `&#182;`). This has the benefit of being a single character that downstream tools can easily replace with `Environment.NewLine` before displaying ... but the negative that a simple read of that string won't "just work".

What would be the pros/cons of simply picking one platform's conventions (for example, will `\n` display correctly on a browser on windows, in an html `<pre>` tag)?

Reply:
https://codepen.io/anon/pen/gGVmag

In this example, I'm writing `document.write("<pre>this is\nanother line</pre>");`. On mac/chrome this outputs:

![image](https://user-images.githubusercontent.com/90380/32016305-9430d52c-b991-11e7-9992-e5959aabd97a.png)

On windows/IE:

![image](https://user-images.githubusercontent.com/90380/32016321-a0e2076e-b991-11e7-9b6f-f20cbc494019.png)

I propose that we standardize on `\n` ... thoughts?

Reply:
@mpMelnikov can you check and see whether, if we use `\n`, and a user copies and pastes that signature ... will the vb.net compiler throw an error, or will it compile successfully?

Reply:
Just like @joelmartinez, I suggest we use `\n` as the standard way of handling new lines - this will be universally acceptable for other reference types and follows standard markup conventions for code docs.

Reply:
> can you check and see whether, if we use \n, and a user copies and pastes that signature ... will the vb.net compiler throw an error, or will it compile successfully?

It works fine. I'm writing `document.write("<pre>Public Structure DocValueType\nImplements IProcess</pre>");`, copy result values from Chrome to Visual Studio or Notepad, the line endings become `\n\r`. It seems to be quite a task to save source line endings while copying text.

@joelmartinez If we decide to use `\n`, maybe we can keep new line after `<Extension()>`? That's how it looks like on [docs.microsoft.com](https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/procedures/extension-methods) now.

Reply:
@mpMelnikov ok, thanks for checking. Yeah, I'm comfortable using `\n`, as long as it works on all platforms :)